imagenet/README.md

```diff
@@ -96,6 +102,7 @@ optional arguments:
                         distributed backend
   --seed SEED           seed for initializing training.
   --gpu GPU             GPU id to use.
+  --xpu set it to 1 for running on XPU, default False for CPU.
```

Comment:
This does not make sense. If you want a flag you can just add a flag. If it's not set, then it's off, if it's set it's on.